jobQueue/setup.js

```diff
@@ -18,19 +18,46 @@ const conf = require('../config');
 const featureToggles = require('feature-toggles');
 const urlParser = require('url');
 const kue = require('kue');
+const activityLogUtil = require('../utils/activityLog');
+
 const redisOptions = {
   redis: conf.redis.instanceUrl.queue,
 };
+
+const express = require('express');
```

Comment:
I don't think /jobQueue is the right place to put this code. We happen to want to do something queue-y on sigterm now, but there will probably be other things that are completely unrelated to job queues. Let's discuss.

Reply:
Also, just a reminder about the other part of this story, with is to enqueue jobs with retry--I'm also ok with making that a separate story if you feel like you're already biting off enough here

Reply:
About enqueue jobs: might be better with another story, it's not clear what is the objective of that.

I agree with not being a better place, coupling queue stuff, and process. Thought it was the best place because is where we create the Job.

Reply:
I have exported a fn from jobQueue to handled shutdown and moved process.once to index (I think it's more generic place to be). From index I am executing jobQueue.gracefulShutdown.

We can in the future create a list of fn to handle SIGTERM and generate an event to execute them one-by-one (I guess by now just with Kue should be ok) - what do you think?